web/web/src/lib/utils/initial.js

```diff
@@ -467,7 +449,132 @@ export const tableColumnTypes = [
         valid: true
       }
     }
-  },
-  { key: 'uuid' },
-  { key: 'binary' }
-]
+  }
+}
+
+export const getParameterizedColumnType = type => {
+  if (Object.keys(parameterizedColumnTypes).includes(type)) {
+    return parameterizedColumnTypes[type]
+  }
+}
+
+const relationalColumnTypeMap = {
+  'lakehouse-iceberg': [
+    'boolean',
+    'integer',
+    'long',
+    'float',
+    'double',
+    'string',
+    'date',
+    'time',
+    'timestamp',
+    'timestamp_tz',
+    'decimal',
+    'fixed',
+    'binary',
+    'uuid'
```

Comment:
You may want to sort this list.

Reply:
I followed the order in the documentation:  
[Lakehouse Iceberg Catalog Table Column Types](https://gravitino.apache.org/docs/0.6.1-incubating/lakehouse-iceberg-catalog/#table-column-types)  

How should I order them, alphabetically?

Reply:
Yup. I'd suggest we sort the list alphabetically if possible (not breaking anything).
Ordered lists have a better maintainability and readability.

Reply:
That's very helpful advice, thank you. I will make the changes.